docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -402,24 +402,21 @@ Prefer negated matchers over `to change.by(0)`.
 [source,ruby]
 ----
 # bad
-expect { run }.to change(Foo, :bar).by(0)
-expect { run }.to change { Foo.bar }.by(0)
-expect { run }
-  .to change(Foo, :bar).by(0)
-  .and change(Foo, :baz).by(0)
-expect { run }
-  .to change { Foo.bar }.by(0)
-  .and change { Foo.baz }.by(0)
+expect { run }.to change(Foo, :x).by(0)
+expect { run }.to change { Foo.x }.by(0)
```

Comment:
Is `bar` -> `x` a necessary change considering we still use `Foo`?

Reply:
I tried to pick something a little shorter to make it one line, but there's no need to make it one line, so I'll revert back to the original.